### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by Rjeey-dev**

### CODE REVIEW

File: php/app/src/Comments/Domain/DTO/Comment.php [hunk 1]
```diff
@@ -18,14 +18,14 @@ class Comment
...
-     * @Serializer\SerializedName("comment")
+     * @Serializer\SerializedName("task_id")
      * @Serializer\Type("string")
      * @Serializer\Groups({
      *     "comments-list",
      *     "comment-detail",
      * })
      */
-    public $comment;
+    public $task_Id;
```
Comment (NinjaBuggs) [inline php/app/src/Comments/Domain/DTO/Comment.php]: $taskId here and everywhere

File: php/app/src/Comments/Domain/Repository/CommentsRepositoryInterface.php [hunk 2]
```diff
@@ -13,7 +13,6 @@ interface CommentsRepositoryInterface
      * @throws CommentNotFoundException
      */
     public function get(CommentId $id): Comment;
-    public function add(Comment $user): void;
-    public function remove(Comment $user): void;
-
+    public function add(Comment $task): void;
```
Comment (NinjaBuggs) [inline php/app/src/Comments/Domain/Repository/CommentsRepositoryInterface.php]: $task ?????

File: php/app/src/Comments/Presentation/Controller/CommentController.php [hunk 3]
```diff
@@ -54,7 +54,7 @@ public function create(Request $request): Response
 
             $command = new CreateCommentCommand(
                 (string)$data['text'],
-                (string)$data['comment']
+                (string)$data['taskId']
```
Comment (NinjaBuggs) [inline php/app/src/Comments/Presentation/Controller/CommentController.php]: should be task_id . From request we should get lowercased and with underscore

File: php/app/src/Tasks/Domain/Entity/Task.php [hunk 4]
```diff
@@ -24,63 +24,54 @@ class Task implements EventRecordableInterface
...
      */
-    private $todo;
+    private $name;
 
     /**
-     * @MongoDB\Field(type="string")
+     * @MongoDB\Field(name="status", type="string")
      */
-    private $doing;
+    private $status;
```
Comment (NinjaBuggs) [inline php/app/src/Tasks/Domain/Entity/Task.php:34]: status should be int 

0 - to do
1 - doing
2 - done

File: php/app/src/Tasks/Application/Command/CreateTaskCommand.php [hunk 5]
```diff
@@ -3,13 +3,12 @@
...
 
-
+use App\Tasks\Domain\Exception\ValidationException;
 use App\Tasks\Domain\ValueObject\TaskId;
 use NinjaBuggs\ServiceBus\Command\CommandInterface;
 
 class CreateTaskCommand implements CommandInterface
 {
-
     private const STATUS_TODO = 0;
```
Comment (Rjeey-dev) [inline php/app/src/Tasks/Application/Command/CreateTaskCommand.php:12]: Why I see an empty line here even though it is not in the code?

**Pull request merged**